src/dosbox.cpp

```diff
@@ -507,12 +507,15 @@ void DOSBOX_Init(void) {
 	pstring->Set_values(force);
 
 #if C_OPENGL
-	Pstring = secprop->Add_path("glshader", Property::Changeable::Always, "sharp");
-	Pstring->Set_help("Path to GLSL shader source to use with OpenGL output (\"none\" to disable).\n"
-	                  "Can be either an absolute path, a file in the \"glshaders\" subdirectory\n"
-	                  "of the DOSBox configuration directory, or one of the built-in shaders:\n"
-	                  "advinterp2x, advinterp3x, advmame2x, advmame3x, rgb2x, rgb3x, scan2x,\n"
-	                  "scan3x, tv2x, tv3x, sharp.");
+	pstring = secprop->Add_path("glshader", always, "sharp");
+	pstring->Set_help(
+	        "Path to GLSL shader source to use with OpenGL output\n"
+	        "(\"none\" to disable).  Can be either an absolute path,\n"
```

Comment:
Minor suggestion: _"Either \"none\" or a GLSL shader name. Only works with OpenGL output. The shader can be an absolute path, a file in ... "_

Reply:
Done :)